examples/platforms/nrf52840/platform-config.h

```diff
@@ -334,4 +336,56 @@
 #define USB_CDC_AS_SERIAL_TRANSPORT  0
 #endif
 
+/*******************************************************************************
+ * @section Radio driver configuration.
+ ******************************************************************************/
+
+/**
+ * @def NRF_DRV_RADIO802154_PENDING_SHORT_ADDRESSES
+ *
+ * Number of slots containing short addresses of nodes for which pending data is stored.
+ *
+ */
+#ifndef NRF_DRV_RADIO802154_PENDING_SHORT_ADDRESSES
+#define NRF_DRV_RADIO802154_PENDING_SHORT_ADDRESSES  OPENTHREAD_CONFIG_MAX_CHILDREN
+#endif
+
+/**
+ * @def NRF_DRV_RADIO802154_PENDING_EXTENDED_ADDRESSES
+ *
+ * Number of slots containing extended addresses of nodes for which pending data is stored.
+ *
+ */
+#ifndef NRF_DRV_RADIO802154_PENDING_EXTENDED_ADDRESSES
+#define NRF_DRV_RADIO802154_PENDING_EXTENDED_ADDRESSES  OPENTHREAD_CONFIG_MAX_CHILDREN
+#endif
+
+/**
+ * @def NRF_RAAL_HFCLK_START
+ *
+ * Macro to request High Frequency Clock start. It may use external driver or OS function.
+ *
+ */
+#ifndef NRF_RAAL_HFCLK_START
```

Comment:
Wouldn't we want to fail hard here (or at least emit a warning) if this was already defined?